server/main.py

```diff
@@ -38,14 +40,13 @@
 app.url_map.strict_slashes = False
 
 @app.route(
-    '/api/recruiter/<int:recruiter_id>/<int:applicant_id>/claim', methods=['GET'])
+    '/api/recruits/claim/<int:applicant_id>', methods=['GET'])
```

Comment:
Just to confirm, only a recruiter themselves should be able to claim an applicant (they can't be assigned one by an admin/senior recruiter)?

Reply:
yes  that's right